src/components/Acknowledgment/Acknowledgment.module.scss

```diff
@@ -0,0 +1,59 @@
+@import "../../styles/variables";
+
+.acknowledgment_container {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  margin-left: 5px;
+
+  .label {
+    text-transform: uppercase;
+    font-weight: 700;
+    color: $blue;
+    margin-right: 5px;
+  }
+
+  .names_container {
+    display: flex;
+
+    .name {
+      background-color: $grey;
+      padding: 5px 7px 7px;
+      margin-right: 5px;
+      border-radius: 10px;
+    }
+
+    a {
+      margin-left: 5px;
+
+      svg {
```

Comment:
I thought this looked better at 1.1rem but I'll leave that for you to make the final call.

Reply:
Are you sure? This is what `margin-left: 1.1rem;` looks like:

![image](https://user-images.githubusercontent.com/75239264/135807909-4431ba00-4526-4690-91c6-0514538c9d4d.png)